pkg/config/toml/defaults/fallback.toml

```
@@ -53,6 +53,7 @@ FeeCapDefault = '100 gwei'
 TipCapDefault = '1'
 TipCapMin = '1'
 EstimateLimit = false
+EstimationSenderAddress = ''
```

Does repeating Estimation add value here?
```suggestion
SenderAddress = ''
```

---

You're correct - I'll update this to SenderAddress instead

---

Actually, on second thought, just naming this "SenderAddress" could lead to a misunderstanding of the config, as this is not the actual sender address of the transaction - it is only the sender address used for gas estimation. I'll change it back to "EstimationSenderAddress"

---

But the fully qualified name is the thing to consider - we are already inside of the `GasEstimator` table here: `EVM.GasEstimator.SenderAddress` vs. `EVM.GasEstimator.EstimationSenderAddress`

---

Thank you Jordan - that makes sense; I've updated the parameter name